asmcli/lib/parse.sh

```diff
@@ -95,6 +95,10 @@ parse_args() {
         context_set-option "ENABLE_ALL" 1
         shift 1
         ;;
+      --use_fleet_api | --use-fleet-api)
```

Comment:
Consider --use-automatic-management as the name of the flag?

Reply:
Discussed offline. Will do this in a future version.